core/src/main/scala/cats/Show.scala

```diff
@@ -25,6 +25,13 @@ object Show {
     def show(a: A): String = a.toString
   }
 
+  def doShow[A](a: A)(implicit ev: Show[A]): String =
```

Comment:
I'm not liking `doShow` either. I wanted to call it simply `show`, but that's taken for the smart constructor:

```scala
def show[A](f: A => String): Show[A]
```

An analogous smart constructor for `Eq` is called [`instance`](https://github.com/typelevel/cats/blob/65433840283da5513a6d3ea5cce082596d9246c9/kernel/src/main/scala/cats/kernel/Eq.scala#L86), so we would only make naming more consistent if we renamed current `show` to `instance` and free up the `show` name for `doShow`.

Otherwise, I could go with `callShow`.